### PR TITLE
runtime.requestUpdateCheck() argument in example correction

### LINKED ISSUE #16798: “runtime.requestUpdateCheck()” article: wrong argument in example
### MDN URL

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/requestUpdateCheck

### What specific section or headline is this issue about?

examples

### What information was incorrect, unhelpful, or incomplete?

let requestingCheck = browser.runtime.requestUpdateCheck(onRequested);

### What did you expect to see?

let requestingCheck = browser.runtime.requestUpdateCheck();

### Do you have any supporting links, references, or citations?

same page:

Parameters: None.

### Do you have anything more you want to share?

this was from using the example code, and received this error:
Uncaught Error: Expected at most 0 arguments for requestUpdateCheck()

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/mozilla/add-ons/webextensions/api/runtime/requestupdatecheck`
* MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/requestUpdateCheck
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/mozilla/add-ons/webextensions/api/runtime/requestupdatecheck/index.md
* Last commit: https://github.com/mdn/content/commit/94d3111ec9d3c77c71486432072eba26e6e221dd
* Document last modified: 2022-05-06T06:20:43.000Z

</details>

### PR DESCRIPTION
**Pull request by rebloor**
### Description

Fixes #16798. Verified on Chrome (as this method isn't supported by Firefox):

image

Also:

- swapped the order of examples and browser compatibility sections
- fixed minor stylistic and grammar issues

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/api/runtime/requestupdatecheck/index.md [hunk 1]
```diff
@@ -23,24 +23,20 @@ None.
 
 ### Return value
 
-A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that will be fulfilled with two arguments:
+A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that is fulfilled with two arguments:
```
Comment (Rob--W) [inline files/en-us/mozilla/add-ons/webextensions/api/runtime/requestupdatecheck/index.md:26]:
```suggestion
A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that is fulfilled with an object with the result of the update request.
```

The previous and current documentation is incorrect. It's a single object, not a number of positional parameters.

File: files/en-us/mozilla/add-ons/webextensions/api/runtime/requestupdatecheck/index.md [hunk 2]
```diff
@@ -23,24 +23,20 @@ None.
...
 
 - `status`
   - : A {{WebExtAPIRef('runtime.RequestUpdateCheckStatus')}} value — the result of the update check.
 - `details` {{optional_inline}}
```
Comment (Rob--W) [inline files/en-us/mozilla/add-ons/webextensions/api/runtime/requestupdatecheck/index.md:30]:
```suggestion
- `result` {{optional_inline}}
  - `status`
    - : A {{WebExtAPIRef('runtime.RequestUpdateCheckStatus')}} value — the result of the update check.
```

Changes:
- moved "status" under the object.
- renamed "details" to "result", consistent with Chrome's docs.

File: files/en-us/mozilla/add-ons/webextensions/api/runtime/requestupdatecheck/index.md [hunk 3]
```diff
@@ -23,24 +23,20 @@ None.
...
-
-{{Compat}}
-
 ## Examples
 
-Request an update, and log the new version if one is available:
+Request an update and log the new version if one is available:
 
 ```js
 function onRequested(status, details) {
```
Comment (Rob--W) [inline files/en-us/mozilla/add-ons/webextensions/api/runtime/requestupdatecheck/index.md:42]:
```suggestion
function onRequested(result) {
```

Github doesn't offer more context, more changes that you need to make before merging:

- Replace `status` with `result.status` (2x)
- Replace `details.version` with `result.version`

**Review by Rob--W — CHANGES_REQUESTED** [review]

**Review by Rob--W — COMMENTED** [review]
The [rendered result](https://pr29715.content.dev.mdn.mozit.cloud/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/requestUpdateCheck) looks very broken. Could you fix up the formatting?

**Comment by rebloor** [comment]
@Rob--W On looking at this again it wasn't just the formatting that was wrong. Fixed

**Review by Rob--W — APPROVED** [review]

**Pull request merged**